neuralmonkey/readers/image_reader.py

```diff
@@ -60,9 +60,46 @@ def load(list_files: List[str]):
     return load
 
 
+def imagenet_reader(prefix: str,
+                    target_width: int=227,
+                    target_height: int=227) -> Callable:
```

Comment:
To tak ty imagenetový sítě mívaj. Nebo taky 224. Uměle si zvětšujou trénovací data tak, že z 230x230 berou o něco menší podčtverce.